scripts/sanitycheck

```diff
@@ -827,17 +837,6 @@ class DeviceHandler(Handler):
         else:
             self.set_state(out_state, handler_time)
 
-        if post_script:
-            with subprocess.Popen(post_script, stderr=subprocess.PIPE, stdout=subprocess.PIPE) as proc:
-                try:
-                    (stdout, stderr) = proc.communicate(timeout=30)
-                    logger.debug(stdout.decode())
-
-                except subprocess.TimeoutExpired:
-                    proc.kill()
-                    (stdout, stderr) = proc.communicate()
```

Comment:
It seems that line 831-838 is same as line 806-813. Can we wrap them in a function, then call the function?

Reply:
Let's make separate PR for this I think.

Reply:
The function being changed is too long and hard to understand, so would be nice to not add to it. I'd do it in this one.